docs/DeveloperGuide.adoc

```diff
@@ -781,14 +781,25 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 == Product Scope
 
 *Target user profile*:
-
+* Freshie Couples/Students in NUS School of Computing
+* - Time Table Sync to NUS Mods in order to find common breaks
+* - Notify on important dates
+* - Journal diary on the meeting
+* - Campus direction shortcuts for new students and when it rains
+* - Couples having diet issues
+* - Location on where to eat
 * has a need to manage a significant number of contacts
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
 *Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Our team visions to help new students to not get lost in their first day of school or when it rains. Because right now, there is no proper channel that helps them to solve this problem.
```

Comment:
by "students" you mean "SoC students", right? Try to be as specific as possible.